src/components/Modals/NewCourse/CourseSelector.vue

```diff
@@ -17,47 +17,44 @@
         :class="['search-result', currentFocus === index ? 'autocomplete-active' : '']"
         @click="selectCourse(matchingCourse)"
       >
-        {{ matchingCourse.title }}
-        <input type="hidden" :value="matchingCourse.title" :name="matchingCourse.roster" />
```

Comment:
I think `<input>` here was for accessibility purposes. Is it ok to remove?

Reply:
I think it's not. It was there back when we have DOM manipulation and we use the input to read out the data we need instead of using emit.
The type is hidden, so the semantics is that it will also be hidden when viewed by the accessibility software, so there should be no impact on a11y.